### ISSUE TITLE
[Bug]: PVC UI tests fails with selenium.common.exceptions.TimeoutException: Message: Failed to find the element (css selector,button[aria-label='Actions'])

### ISSUE THREAD

**Issue by sidhant-agrawal** [red-hat-storage/ocs-ci#12389]
### Description
PVC UI tests fails with selenium.common.exceptions.TimeoutException: Message: Failed to find the element (css selector,button[aria-label='Actions'])

Error message:
```
_locator = ("button[aria-label='Actions']", 'css selector'), _timeout = 30
_enable_screenshot = False, _copy_dom = False


def _do_click(_locator, _timeout=30, _enable_screenshot=False, _copy_dom=False):
    # wait for page fully loaded only if an element was not located
    # prevents needless waiting and frequent crushes on ODF Overview page,
    # when metrics and alerts frequently updated
    if not self.get_elements(_locator):
        self.page_has_loaded()
    screenshot = (
        ocsci_config.UI_SELENIUM.get("screenshot") and enable_screenshot
    )
    if screenshot:
        self.take_screenshot()
    if _copy_dom:
        self.copy_dom()

    wait = WebDriverWait(self.driver, timeout)
    try:
        if (
            version.get_semantic_version(get_ocp_version(), True)
            <= version.VERSION_4_11
        ):
            element = wait.until(
                ec.element_to_be_clickable((locator[1], locator[0]))
            )
        else:

          element = wait.until(
                ec.visibility_of_element_located((locator[1], locator[0]))
            )

ocs_ci/ocs/ui/base_ui.py:222: 



self = <selenium.webdriver.support.wait.WebDriverWait (session="510e985eb471505ee1050e7e47546d04")>
method = <selenium.webdriver.support.expected_conditions.visibility_of_element_located object at 0x7fb469e7c7c0>
message = ''


def until(self, method, message=''):
    """Calls the method provided with the driver as an argument until the \
    return value is not False."""
    screen = None
    stacktrace = None

    end_time = time.time() + self._timeout
    while True:
        try:
            value = method(self._driver)
            if value:
                return value
        except self._ignored_exceptions as exc:
            screen = getattr(exc, 'screen', None)
            stacktrace = getattr(exc, 'stacktrace', None)
        time.sleep(self._poll)
        if time.time() > end_time:
            break

  raise TimeoutException(message, screen, stacktrace)
E       selenium.common.exceptions.TimeoutException: Message:



venv/lib64/python3.9/site-packages/selenium/webdriver/support/wait.py:80: TimeoutException


During handling of the above exception, another exception occurred:


self = <tests.functional.ui.test_pvc_ui.TestPvcUserInterface object at 0x7fb4498fe970>
project_factory = <function project_factory_fixture..factory at 0x7fb44939d940>
teardown_factory = <function teardown_factory_fixture..factory at 0x7fb44939d5e0>
setup_ui_class_factory = <function setup_ui_class_factory..factory at 0x7fb46842f1f0>
sc_name = 'ocs-external-storagecluster-cephfs', access_mode = 'ReadWriteMany'
pvc_size = '2', vol_mode = 'Filesystem'


@tier1
@pytest.mark.parametrize(
    argnames=["sc_name", "access_mode", "pvc_size", "vol_mode"],
    argvalues=[
        pytest.param(
            "ocs-storagecluster-cephfs",
            "ReadWriteMany",
            "2",
            "Filesystem",
            marks=pytest.mark.polarion_id("OCS-5210"),
        ),
        pytest.param(
            "ocs-storagecluster-ceph-rbd",
            "ReadWriteMany",
            "3",
            "Block",
            marks=pytest.mark.polarion_id("OCS-5211"),
        ),
        pytest.param(
            "ocs-storagecluster-cephfs",
            "ReadWriteOnce",
            "10",
            "Filesystem",
            marks=pytest.mark.polarion_id("OCS-5212"),
        ),
        pytest.param(
            *["ocs-storagecluster-ceph-rbd", "ReadWriteOnce", "11", "Block"],
            marks=[skipif_ocs_version("<4.7"), pytest.mark.polarion_id("OCS-5206")],
        ),
        pytest.param(
            "ocs-storagecluster-ceph-rbd",
            "ReadWriteOnce",
            "13",
            "Filesystem",
            marks=pytest.mark.polarion_id("OCS-5207"),
        ),
    ],
)
def test_create_resize_delete_pvc(
    self,
    project_factory,
    teardown_factory,
    setup_ui_class_factory,
    sc_name,
    access_mode,
    pvc_size,
    vol_mode,
):
    """
    Test create, resize and delete pvc via UI

    """

    setup_ui_class_factory()

    # Creating a test project via CLI
    pro_obj = project_factory()
    project_name = pro_obj.namespace

    pvc_ui_obj = PvcUI()

    # Creating PVC via UI
    pvc_name = create_unique_resource_name("test", "pvc")

    if config.DEPLOYMENT["external_mode"]:
        if sc_name == constants.CEPHFILESYSTEM_SC:
            sc_name = constants.DEFAULT_EXTERNAL_MODE_STORAGECLASS_CEPHFS
        elif sc_name == constants.CEPHBLOCKPOOL_SC:
            sc_name = constants.DEFAULT_EXTERNAL_MODE_STORAGECLASS_RBD

    pvc_ui_obj.create_pvc_ui(
        project_name, sc_name, pvc_name, access_mode, pvc_size, vol_mode
    )

    pvc_objs = get_all_pvc_objs(namespace=project_name)
    pvc = [pvc_obj for pvc_obj in pvc_objs if pvc_obj.name == pvc_name]

    assert pvc[0].size == int(pvc_size), (
        f"size error| expected size:{pvc_size} \n "
        f"actual size:{str(pvc[0].size)}"
    )

    assert pvc[0].get_pvc_access_mode == access_mode, (
        f"access mode error| expected access mode:{access_mode} "
        f"\n actual access mode:{pvc[0].get_pvc_access_mode}"
    )

    assert pvc[0].backed_sc == sc_name, (
        f"storage class error| expected storage class:{sc_name} "
        f"\n actual storage class:{pvc[0].backed_sc}"
    )

    assert pvc[0].get_pvc_vol_mode == vol_mode, (
        f"volume mode error| expected volume mode:{vol_mode} "
        f"\n actual volume mode:{pvc[0].get_pvc_vol_mode}"
    )

    # Verifying PVC via UI
    logger.info("Verifying PVC Details via UI")
    pvc_ui_obj.verify_pvc_ui(
        pvc_size=pvc_size,
        access_mode=access_mode,
        vol_mode=vol_mode,
        sc_name=sc_name,
        pvc_name=pvc_name,
        project_name=project_name,
    )
    logger.info("PVC Details Verified via UI..!!")

    # Creating Pod via CLI
    logger.info("Creating Pod")
    if sc_name in constants.DEFAULT_STORAGECLASS_RBD:
        interface_type = constants.CEPHBLOCKPOOL
    elif sc_name in constants.DEFAULT_EXTERNAL_MODE_STORAGECLASS_RBD:
        interface_type = constants.CEPHBLOCKPOOL
    else:
        interface_type = constants.CEPHFILESYSTEM

    new_pod = helpers.create_pod(
        interface_type=interface_type,
        pvc_name=pvc_name,
        namespace=project_name,
        raw_block_pv=vol_mode == constants.VOLUME_MODE_BLOCK,
    )

    logger.info(f"Waiting for Pod: state= {constants.STATUS_RUNNING}")
    wait_for_resource_state(
        resource=new_pod, state=constants.STATUS_RUNNING, timeout=120
    )

    # Calling the Teardown Factory Method to make sure Pod is deleted
    teardown_factory(new_pod)

    # Expanding the PVC
    logger.info("Pvc Resizing")
    new_size = int(pvc_size) + 3

  pvc_ui_obj.pvc_resize_ui(
        pvc_name=pvc_name, new_size=new_size, project_name=project_name
    )

tests/functional/ui/test_pvc_ui.py:169: 



ocs_ci/ocs/ui/pvc_ui.py:164: in pvc_resize_ui
    self.do_click(self.pvc_loc["pvc_actions"])
ocs_ci/ocs/ui/base_ui.py:235: in do_click
    _do_click(locator, timeout, enable_screenshot, copy_dom)



_locator = ("button[aria-label='Actions']", 'css selector'), _timeout = 30
_enable_screenshot = False, _copy_dom = False


def _do_click(_locator, _timeout=30, _enable_screenshot=False, _copy_dom=False):
    # wait for page fully loaded only if an element was not located
    # prevents needless waiting and frequent crushes on ODF Overview page,
    # when metrics and alerts frequently updated
    if not self.get_elements(_locator):
        self.page_has_loaded()
    screenshot = (
        ocsci_config.UI_SELENIUM.get("screenshot") and enable_screenshot
    )
    if screenshot:
        self.take_screenshot()
    if _copy_dom:
        self.copy_dom()

    wait = WebDriverWait(self.driver, timeout)
    try:
        if (
            version.get_semantic_version(get_ocp_version(), True)
            <= version.VERSION_4_11
        ):
            element = wait.until(
                ec.element_to_be_clickable((locator[1], locator[0]))
            )
        else:
            element = wait.until(
                ec.visibility_of_element_located((locator[1], locator[0]))
            )
        element.click()
    except TimeoutException as e:
        self.take_screenshot()
        self.copy_dom()
        logger.error(e)

      raise TimeoutException(
            f"Failed to find the element ({locator[1]},{locator[0]})"
        )
E           selenium.common.exceptions.TimeoutException: Message: Failed to find the element (css selector,button[aria-label='Actions'])

ocs_ci/ocs/ui/base_ui.py:230: TimeoutException
```

RP link: https://url.corp.redhat.com/af1e2d5
